csharp/ql/lib/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -2425,6 +2449,21 @@ private predicate readContentStep(Node node1, Content c, Node node2) {
   VariableCapture::readStep(node1, c, node2)
 }
 
+/**
+ * Holds if data can flow from `node1` to `node2` via an assignment to
+ * the content set `c` of a delegate call.
+ *
+ * If there is a delegate call f(x), then we read the return of the delegate
+ * call.
+ */
+private predicate readStepDelegateCall(Node node1, ContentSet c, Node node2) {
```

Comment:
Again, because of CFG splitting:
```ql
private predicate readStepDelegateCall(Node node1, ContentSet c, OutNode node2) {
  exists(ExplicitDelegateLikeDataFlowCall call |
    lambdaCall(call, _, node1) and
    node2.getCall(TNormalReturnKind()) = call and
    c.isDelegateCallReturn()
  )
}
```